src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+body {
+  margin: 0;
+  font-family: $main-font;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card {
+  padding: 16px;
+  padding-top: 32px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: $main-white-color;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+
+  background-image: url(../images/imac.jpeg);
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  align-self: flex-start;
+  color: $main-color-text;
+  margin: 0;
+  margin-bottom: 16px;
+}
+
+.card__assessment {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__price {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  color: $main-color-text;
+}
+
+.card__price-cost {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  box-sizing: border-box;
+  width: 100%;
+  height: 40px;
+  font-family: $main-font;
+  color: $main-white-color;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  background-color: $color-button;
+  border-radius: 5px;
+  border: 1px solid $color-button;
+  outline: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-transform: uppercase;
+  text-align: center;
+}
+
+.card:hover {
+  transform: scale(1.2);
+  transition-duration: 300ms;
+}
+
+.card:hover .card__title {
+  color: #34568b;
+}
+
+.card__button:hover {
+  background-color: $main-white-color;
+  color: $color-button;
+}
```

Comment:
Ensure that the hover effect on the `.card__button` also has a transition duration of 300ms to maintain consistency with other hover effects.